src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -9,9 +9,6 @@
      -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(SignAssembly)'=='true'">
```

Comment:
This needs to be deleted from `SonarScanner.MSBuild.TFS.csproj` and `SonarScanner.MSBuild.TFS.csproj`

Reply:
I guess it wanted to be `SonarScanner.MSBuild.TFS.csproj` and `SonarScanner.MSBuild.TFS.Classic.csproj`